rust/agama-dbus-server/src/network/system.rs

```diff
@@ -16,7 +16,7 @@ pub struct NetworkSystem<T: Adapter> {
 
 impl<T: Adapter> NetworkSystem<T> {
     pub fn new(conn: zbus::Connection, adapter: T) -> Self {
-        let (actions_tx, actions_rx) = unbounded();
+        let (actions_tx, actions_rx) = mpsc::channel(100);
```

Comment:
what is that `100`? Buffer size?

Reply:
Yes, sorry, I should have used an unbounded channel. I have adjusted the PR.